### DISCUSSION TITLE
How do I change my path to store data?

### DISCUSSION THREAD

**Discussion by haozhuang0000** [milvus-io/milvus#34182] [Q&A and General discussion] [answered]
Currently my milvus is build under path : /home/user
I want to migrate my milvus database to : /data/user

How can I do this?

**Comment by yhmo** [comment]
In which way you deployed the milvus?
By docker? https://milvus.io/docs/install_standalone-docker.md
By docker-compose? https://milvus.io/docs/install_standalone-docker-compose.md

**Reply by haozhuang0000** [reply]
> In which way you deployed the milvus? By docker? https://milvus.io/docs/install_standalone-docker.md By docker-compose? https://milvus.io/docs/install_standalone-docker-compose.md

Hi @yhmo , it is by docker. https://milvus.io/docs/install_standalone-docker.md

**Comment by yhmo** [comment, marked as answer]
Follow these steps:
1. Assume you put the  "standalone_embed.sh" under this path **/home/user/test**,  after you run the command `bash standalone_embed.sh start`, you will see a new file "embedEtcd.yaml" and a new subdirectory named "volumes" under this path:
![Screenshot from 2024-06-26 17-10-11](https://github.com/milvus-io/milvus/assets/2282099/ffcf12cf-e632-4e10-8a5d-7a9ec6ad0c13)

2. Stop the milvus by command `bash standalone_embed.sh stop`
3. Use the command "docker ps -a" to list all the docker containers, you will see a container named "milvus-standalone"
4. Use the command "docker rm [container_id]" to delete the "milvus-standalone" container
5. Copy the entire directory **/home/user/test** to **/data/user/test**
6. Under the **/data/user/test**, run command `bash standalone_embed.sh start` again. You will see a new milvus standalone container started, your data is in this container.

**Reply by haozhuang0000** [reply]
> Follow these steps:
> 
> 1. Assume you put the  "standalone_embed.sh" under this path **/home/user/test**,  after you run the command `bash standalone_embed.sh start`, you will see a new file "embedEtcd.yaml" and a new subdirectory named "volumes" under this path:
>    ![Screenshot from 2024-06-26 17-10-11](https://private-user-images.githubusercontent.com/2282099/343090055-ffcf12cf-e632-4e10-8a5d-7a9ec6ad0c13.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0NTUxNzgsIm5iZiI6MTcxOTQ1NDg3OCwicGF0aCI6Ii8yMjgyMDk5LzM0MzA5MDA1NS1mZmNmMTJjZi1lNjMyLTRlMTAtOGE1ZC03YTllYzZhZDBjMTMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjdUMDIyMTE4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmVlNjQyNzhjMWFhN2Y1OGRlYTFjZDQ2MjJiMjlkZmJlMWJiZjdjMWU5Y2VlYWM4ZTU3M2M0YzZhN2M5ODk2ZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.M7_ElAIXbK9_2_z-IJ9r8JnxAmiNFN1Vr5iO7CKfSxw)
> 2. Stop the milvus by command `bash standalone_embed.sh stop`
> 3. Use the command "docker ps -a" to list all the docker containers, you will see a container named "milvus-standalone"
> 4. Use the command "docker rm [container_id]" to delete the "milvus-standalone" container
> 5. Copy the entire directory **/home/user/test** to **/data/user/test**
> 6. Under the **/data/user/test**, run command `bash standalone_embed.sh start` again. You will see a new milvus standalone container started, your data is in this container.

Dear @yhmo ,Thanks for your reply! I followed the steps that you provided.
After I moved my files to /data/user/test, and I restarted milvus, and it stuck here for very long
![image](https://github.com/milvus-io/milvus/assets/71269145/7e84f857-f4fe-4c37-8662-e9436abc2964)

**Reply by yhmo** [reply]
If you have a large collection that is loaded in memory, after restart, milvus will automatically load it. A large collection will take long time to load.
If you don't have a large collection, use **docker ps -a** to observe the milvus container ID, then **docker logs -f [container_id]** to print the log. There might be WARN/ERROR in the log.

**Comment by haozhuang0000** [comment]
@yhmo Thanks for helping! Very clear instruction on how to move milvus db to another path
